src/js/abstract/base-component.js

```diff
@@ -355,7 +355,7 @@ export default class BaseComponent extends HTMLElement {
     if (this.contextNode) {
       clearTimeout(this.timeoutId);
       this.timeoutId = setTimeout(() => {
-        if (ENV !== 'production') {
+        if (ENV !== 'PROD') {
```

Comment:
here can we not import the constanst file cause is a node export? mmm not very nice. maybe better option to duplicate the constanst file parallel to the original one in a folder and name them: constanst.node and constant.es6